### PR TITLE
lcow: fix debug in startServiceVMIfNotRunning()

### PR DESCRIPTION
**Pull request by kolyshkin**
When go-1.11beta1 is used for building, the following error is reported:

> 14:56:20 daemon\graphdriver\lcow\lcow.go:236: Debugf format %s reads arg #2, but call has 1 arg

While fixing this, let's also fix a few other things in this very function (`startServiceVMIfNotRunning()`):

1. Do not use `fmt.Printf` when not required.
2. Use `title` whenever possible.
3. Don't add `id` to messages as `title` already has it.
4. Remove duplicated colons.
5. Try to unify style of messages.
6. `s/startservicevmifnotrunning/startServiceVMIfNotRunning/`
...

In general, logging/debugging here is a mess and requires much more love than I can give it at the moment. Ideas for improvement:

1. Add a global `var logger = logrus.WithField("storage-driver", "lcow")` and use it everywhere else in the code.
2. Use `logger.WithField("id", id)` whenever possible (same for "context" and other similar fields).
3. Revise all the errors returned to be uniform.
4. Make use of `errors.Wrap[f]` whenever possible.

### CODE REVIEW

**Comment by kolyshkin** [comment]
@johnstep @￼ jhowardmsft PTAL

**Comment by kolyshkin** [comment]
This is separated from https://github.com/moby/moby/pull/37358

**Comment by jhowardmsft** [comment]
Sure. LGTM

**Review by lowenna — APPROVED** [review]

**Comment by kolyshkin** [comment]
z and ppc failures are obviously unrelated (some glitch in java).

File: daemon/graphdriver/lcow/lcow.go [hunk 1]
```diff
@@ -219,21 +219,21 @@ func (d *Driver) startServiceVMIfNotRunning(id string, mvdToAdd []hcsshim.Mapped
 	// Use the global ID if in global mode
 	id = d.getVMID(id)
 
-	title := fmt.Sprintf("lcowdriver: startservicevmifnotrunning %s:", id)
+	title := "lcowdriver: startServiceVMIfNotRunning " + id
```
Comment (thaJeztah) [inline daemon/graphdriver/lcow/lcow.go]: Perhaps for a follow up, but we should `logrus.WithFields` for this information (similar to what we do for other storage drivers)

File: daemon/graphdriver/lcow/lcow.go [hunk 2]
```diff
@@ -219,21 +219,21 @@ func (d *Driver) startServiceVMIfNotRunning(id string, mvdToAdd []hcsshim.Mapped
...
-	logrus.Debugf("%s: Adding entry to service vm map", title)
+	logrus.Debugf("%s: adding entry to service vm map", title)
 	svm, exists, err := d.serviceVms.add(id)
 	if err != nil && err == errVMisTerminating {
 		// VM is in the process of terminating. Wait until it's done and and then try again
-		logrus.Debugf("%s: VM with current ID still in the process of terminating: %s", title, id)
+		logrus.Debugf("%s: VM with current ID still in the process of terminating", title)
 		if err := svm.getStopError(); err != nil {
-			logrus.Debugf("%s: VM %s did not stop successfully: %s", title, id, err)
+			logrus.Debugf("%s: VM did not stop successfully: %s", title, err)
```
Comment (thaJeztah) [inline daemon/graphdriver/lcow/lcow.go]: `logrus.WithError(err).Debug(....)`

**Review by thaJeztah — APPROVED** [review]
LGTM, but left some suggestions; I'm ok to do that in a follow up (that is, of course, if you agree) or as part of this PR; let me know

**Review by johnstep — APPROVED** [review]

File: daemon/graphdriver/lcow/lcow.go (hunk 1 above)
Comment (kolyshkin) [reply daemon/graphdriver/lcow/lcow.go]: I concur (see the patch description).

File: daemon/graphdriver/lcow/lcow.go (hunk 2 above)
Comment (kolyshkin) [reply daemon/graphdriver/lcow/lcow.go]: ditto

File: daemon/graphdriver/lcow/lcow.go (hunk 1 above)
Comment (kolyshkin) [reply daemon/graphdriver/lcow/lcow.go]: ...but I'd rather do it in a separate PR later as this fix is quite big already -- and initially it supposed to be a one-liner :rofl:

Comment (thaJeztah) [reply daemon/graphdriver/lcow/lcow.go]: D'oh!! Didn't read that far into your description 🤦‍♂️☺️😝

**Comment by thaJeztah** [comment]
PowerPC https://jenkins.dockerproject.org/job/Docker-PRs-powerpc/10507/console failure is unrelated, and tracked through https://github.com/moby/moby/issues/37408


```
01:09:07 FAIL: docker_api_swarm_service_test.go:311: DockerSwarmSuite.TestAPISwarmServicesFailedUpdate
01:09:07 
01:09:07 [d609c07d09b2c] waiting for daemon to start
01:09:07 [d609c07d09b2c] daemon started
01:09:07 
01:09:07 [d0e6e563a200d] waiting for daemon to start
01:09:07 [d0e6e563a200d] daemon started
01:09:07 
01:09:07 [d6ea22a0c99a1] waiting for daemon to start
01:09:07 [d6ea22a0c99a1] daemon started
01:09:07 
01:09:07 docker_api_swarm_service_test.go:347:
01:09:07     waitAndAssert(c, defaultReconciliationTimeout, daemons[0].CheckRunningTaskImages, checker.DeepEquals,
01:09:07         map[string]int{image1: instances})
01:09:07 docker_utils_test.go:435:
01:09:07     c.Assert(v, checker, args...)
01:09:07 ... obtained map[string]int = map[string]int{"busybox:latest":4}
01:09:07 ... expected map[string]int = map[string]int{"busybox:latest":5}
01:09:07 
01:09:07 [d609c07d09b2c] exiting daemon
01:09:07 [d0e6e563a200d] exiting daemon
01:09:07 [d6ea22a0c99a1] exiting daemon
```

**Pull request merged**